packages/crayons-core/src/components/form-control/form-control.tsx

```diff
@@ -418,6 +419,41 @@ export class FormControl {
           );
         }
         break;
+      case 'FILES':
+        {
+          const multiple = this.fieldProps?.multiple ? true : false;
+          const errors =
+            this.touched && this.error
```

Comment:
In the render function below, we are already showing the error message like `field is required`. 
Do we need to pass the errors and show it inside the file uploader component as well?. 

if we add a prop like `state` and set it to error, we can color the border to `red` .

Reply:
The error messages in render was only for slots. Else, it is usually sent as a prop to the form element, which we have handled in file uploader.